### PR TITLE
MS Exchange Server - modeling rule

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XDR-52866

## Description
added the modeling rule of MS Exchange Server.

### CODE REVIEW

File: Packs/MicrosoftExchangeServer/ModelingRules/MicrosoftExchangeServerModeling/MicrosoftExchangeModeling.xif [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[MODEL: dataset="filestream_filestream_raw", model="Email"]
```
Comment (evisochek) [inline Packs/MicrosoftExchangeServer/ModelingRules/MicrosoftExchangeServerModeling/MicrosoftExchangeModeling.xif:1]: need to change to dataset="msft_exchange_raw"

**Comment by evisochek** [comment]
Need to add readme with the XDRC configuration.

File: Packs/MicrosoftExchangeServer/ModelingRules/MicrosoftExchangeServerModeling/MicrosoftExchangeModeling.xif (hunk 1 above)
Comment (tomer-pan) [reply Packs/MicrosoftExchangeServer/ModelingRules/MicrosoftExchangeServerModeling/MicrosoftExchangeModeling.xif]: Done

**Comment by tomer-pan** [comment]
@evisochek  Readme was added

**Review by evisochek — APPROVED** [review]

**Pull request merged**